package.json

```
@@ -53,6 +53,7 @@
     "@semantic-release/github": "7.2.0",
     "@types/doubleclick-gpt": "^2019041801.0.4",
     "@types/jest": "^26.0.20",
+		"@types/google.analytics": "^0.0.41",
```

I think the linting rules should be applied, here for example we have tabs instead of spaces.

```suggestion
    "@types/google.analytics": "^0.0.41",
```